src/appointment/appointment.controller.ts

```diff
@@ -0,0 +1,83 @@
+import {
+    Controller,
+    Get,
+    Param,
+    Post,
+    Body,
+    Patch,
+    UseGuards,
+} from '@nestjs/common';
+import { AppointmentService } from './appointment.service';
+import { UserId } from 'src/decorators/user-id.decorator';
+import { AppointmentDTO } from './appointment.dto';
+import { AppointmentStatus } from 'src/enum/appointment.enum';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { AuthGuard } from 'src/guards/auth.guard';
+
+@ApiBearerAuth()
+@ApiTags('Appointment')
+@UseGuards(AuthGuard)
+@Controller('appointment')
+export class AppointmentController {
+    constructor(private readonly service: AppointmentService) {}
+
+    @Post('create')
+    createAppointment(
+        @Body() appointment: AppointmentDTO,
+        @UserId() studentId: string,
+    ) {
+        return this.service.createAppointment(appointment, studentId);
+    }
+
+    @Get()
+    getAllAppointments() {
+        return this.service.getAllAppointments();
+    }
```

Comment:
The functions' name should follow the pattern in other files too. Please fix them.